app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+  def show
+      @user = User.find_by id: params[:id]
+  end
+
+  def new
+      @user = User.new
+  end
+  
+  def create
+      @user = User.new user_params
+      
+      if @user.save
+          flash[:success] = t ".welcome"
+          redirect_to@user
+      else
+        render :new
+      end
+  end
+  
+  private
+  def user_params
+      params.require(:user).permit(:name, :email, :password, :password_confirmation)
```

Comment:
in 'user_model' file, create constant USER_PARAMS = %i(name email ...).freeze
```suggestion
      params.require(:user).permit USER::USER_PARAMS
```

Reply:
do you mean app/models/user.rb